openedx/core/djangoapps/notifications/management/commands/delete_notifications.py

```diff
@@ -0,0 +1,70 @@
+"""
+Management command for deleting notifications with parameters
+"""
+import datetime
+from django.core.management.base import BaseCommand
+
+from openedx.core.djangoapps.notifications.base_notification import (
+    COURSE_NOTIFICATION_APPS,
+    COURSE_NOTIFICATION_TYPES
+)
+from openedx.core.djangoapps.notifications.tasks import delete_notifications
+
+
+class Command(BaseCommand):
+    """
+    Invoke with:
+
+        python manage.py lms delete_notifications
+    """
+    help = (
+        "Delete notifications that meets a criteria. Requires app_name, notification_type and created (duration)"
+    )
+
+    def add_arguments(self, parser):
+        """
+        Add command line arguments to management command
+        """
+        parser.add_argument('--app_name', choices=list(COURSE_NOTIFICATION_APPS.keys()), required=True)
+        parser.add_argument('--notification_type', choices=list(COURSE_NOTIFICATION_TYPES.keys()),
+                            required=True)
+        parser.add_argument('--created', type=argparse_date, required=True,
+                            help="Allowed date formats YYYY-MM-DD. YYYY Year. MM Month. DD Date."
+                                 "Duration can be specified with ~. Maximum 15 days duration is allowed")
+        parser.add_argument('--course_id', required=False)
+
+    def handle(self, *args, **kwargs):
+        """
+        Calls delete notifications task
+        """
+        delete_notifications.delay(kwargs)
```

Comment:
it would be nice if there was immediate output (error/success) after the command is run. something like `Deletion task is in progress please check logs to verify`

Reply:
Added log